index/field/docs_field.go

```diff
@@ -149,7 +149,7 @@ type docsField struct {
 	bf     CloseableBoolField
 	intf   CloseableIntField
 	df     CloseableDoubleField
-	sf     CloseableStringField
+	sf     CloseableBytesField
```

Comment:
rename to `bf`

Reply:
There is already a `bf` for the bool types.